.coveragerc

```diff
@@ -0,0 +1,8 @@
+[run]
+omit =
+    # Python specific path
+    /usr/local/lib/python2.7/*
```

Comment:
I did not see a similar problem on AppVeyor. The codecov report from PR #602 looks fine.